weave/tests/trace/test_tracing_resilience.py

```diff
@@ -31,104 +31,104 @@ def test_resilience_to_user_code_errors(client):
     def do_test():
         @weave.op
         def throws():
-            raise TestException("This is a test exception")
+            raise DummyTestException("This is a test exception")
 
         throws()
 
     # The user's exception should be raised - even if we're capturing errors
-    with pytest.raises(TestException):
-        do_test()
+    with raise_on_captured_errors(True):
+        with pytest.raises(DummyTestException):
+            do_test()
 
     # The user's exception should be raised - even if we're not capturing errors
-    with raise_on_captured_errors(False):
-        with pytest.raises(TestException):
-            do_test()
+    with pytest.raises(DummyTestException):
+        do_test()
```

Comment:
This is a test about testing, right?  It seems like it should be in its own file noting so

Reply:
No, it is a test about trace resilience